dist/.magento.env.yaml

```diff
@@ -262,6 +262,20 @@
 #                elasticsearch_index_prefix: magento                                                                  #
 #                elasticsearch_server_timeout: '15'                                                                   #
 #######################################################################################################################
+# ELASTIC_SUITE_CONFIGURATION - use this variable to retain customized ElasticSearch service settings                 #
+# between deployments.                                                                                                #
+# Magento Version: 2.1.4 and later                                                                                    #
```

Comment:
ElasticSuite is only for 2.2 and 2.3 Magento versions

Reply:
Thanks